docs/configuring.md

```diff
@@ -12,26 +12,54 @@ There are two approaches to configuring Pixeebot:
 2. **Global repository configuration:**
    Alternatively, you can create the `pixeebot.yaml` file in the `.github` directory of your `.github` repository. This will serve as a global configuration that applies to multiple repositories.
 
-## YAML
+## Properties
+
+### `ai`
 
-A typical `.yaml` configuration file might look like this:
+Contains settings related to AI functionality.
+
+#### `allow_llm_access`
+
+Setting to `false` disables Pixeebot features that [rely on generative AI](faqs.md) to analyze your code.
+
+> **Note** Pixeebot uses generative AI by default (`true` by default)
```

Comment:
I would move this line to either be first in this section, or last. (If first, can remove the markdown and make it the first line in that initial paragraph.) Right now it feels a bit disconnected because we talk about false, then true, then show an example of false.

Reply:
I struggled a bit with this. In the current version of the docs, we talk about what setting this AI proeprty to `true` does, then at the end of this section we mention "oh by the way, `true` is the default". It didn't sit right with me, because it sounds like we're saying "if you do this, then this will happen, but actually that's the default behavior, so if you don't want that then do the opposite of everything I just explained".

So I tried to rewrite it to say "take this action to change the default behavior, and this is the effect". I think your recommendation to simply move the "`true` is the default" to be first in the section gets us there. I'm going to make that change now.